src/Features/Core/Portable/ExtractInterface/AbstractExtractInterfaceService.cs

```diff
@@ -46,7 +45,7 @@ internal abstract class AbstractExtractInterfaceService : ILanguageService
 
         internal abstract bool ShouldIncludeAccessibilityModifier(SyntaxNode typeNode);
 
-        public async Task<ImmutableArray<ExtractInterfaceCodeAction>> GetExtractInterfaceCodeActionAsync(Document document, TextSpan span, CancellationToken cancellationToken)
+        public async Task<ImmutableArray<ExtractInterfaceCodeAction>> GetExtractInterfaceCodeAction(Document document, TextSpan span, CancellationToken cancellationToken)
```

Comment:
I am not sure why you removed Async? the method still has async and await in it?

Reply:
Woops, that got renamed in the mass purging of Async changes I did. It should be `GetExtractInterfaceCodeActionAsync`